packages/react/package.json

```diff
@@ -115,8 +115,8 @@
     "@mantine/notifications": "^7.0.0",
     "@medplum/core": "4.0.4",
     "@medplum/react-hooks": "4.0.4",
-    "react": "^17.0.2 || ^18.0.0",
-    "react-dom": "^17.0.2 || ^18.0.0",
+    "react": "^17.0.2 || ^18.0.0 || ^19.0.0",
```

Comment:
Are we sure 17 still works with Mantine? If they requiring 18 or 19? 🤔

Reply:
Oh I just read description. We can leave it for now I guess 👍

Reply:
Yeah.... We should probably treat that as a separate issue.

In a perfect world, we would have cut React 17 support in Medplum v4.  Oops.